README.md

```diff
@@ -1,165 +1,77 @@
+> - If you need more time to migrate, you can pin your environment to the latest pre-v0.1.0 release.
+
 # Xee: Xarray + Google Earth Engine
 
 ![Xee Logo](https://raw.githubusercontent.com/google/Xee/main/docs/xee-logo.png)
 
-_An Xarray extension for Google Earth Engine._
+Xee is an Xarray backend for Google Earth Engine. Open `ee.Image` / `ee.ImageCollection` objects as lazy `xarray.Dataset`s and analyze petabyte‑scale Earth data with the scientific Python stack.
 
 [![image](https://img.shields.io/pypi/v/xee.svg)](https://pypi.python.org/pypi/xee)
 [![image](https://static.pepy.tech/badge/xee)](https://pepy.tech/project/xee)
-[![Conda
-Recipe](https://img.shields.io/badge/recipe-xee-green.svg)](https://github.com/conda-forge/xee-feedstock)
+[![Conda Recipe](https://img.shields.io/badge/recipe-xee-green.svg)](https://github.com/conda-forge/xee-feedstock)
 [![image](https://img.shields.io/conda/vn/conda-forge/xee.svg)](https://anaconda.org/conda-forge/xee)
-[![Conda
-Downloads](https://img.shields.io/conda/dn/conda-forge/xee.svg)](https://anaconda.org/conda-forge/xee)
-
-## How to use
+[![Conda Downloads](https://img.shields.io/conda/dn/conda-forge/xee.svg)](https://anaconda.org/conda-forge/xee)
 
-Install with pip:
+## Install
 
-```shell
+```bash
 pip install --upgrade xee
 ```
 
-Install with conda:
+or
 
-```shell
+```bash
 conda install -c conda-forge xee
 ```
 
-Then, authenticate Earth Engine:
-
-```shell
-earthengine authenticate --quiet
-```
-
-Now, in your Python environment, make the following imports:
-
-```python
-import ee
-import xarray
-```
-
-Next, specify your EE-registered cloud project ID and initialize the EE client
-with the high volume API:
-
-```python
-ee.Initialize(
-    project='my-project-id'
-    opt_url='https://earthengine-highvolume.googleapis.com')
-```
-
-Open any Earth Engine ImageCollection by specifying the Xarray engine as `'ee'`:
-
-```python
-ds = xarray.open_dataset('ee://ECMWF/ERA5_LAND/HOURLY', engine='ee')
-```
-
-Open all bands in a specific projection (not the Xee default):
-
-```python
-ds = xarray.open_dataset('ee://ECMWF/ERA5_LAND/HOURLY', engine='ee',
-                         crs='EPSG:4326', scale=0.25)
-```
-
-Open an ImageCollection (maybe, with EE-side filtering or processing):
+## Minimal example
 
 ```python
-ic = ee.ImageCollection('ECMWF/ERA5_LAND/HOURLY').filterDate(
-    '1992-10-05', '1993-03-31')
-ds = xarray.open_dataset(ic, engine='ee', crs='EPSG:4326', scale=0.25)
-```
-
-Open an ImageCollection with a specific EE projection or geometry:
-
-```python
-ic = ee.ImageCollection('ECMWF/ERA5_LAND/HOURLY').filterDate(
-    '1992-10-05', '1993-03-31')
-leg1 = ee.Geometry.Rectangle(113.33, -43.63, 153.56, -10.66)
-ds = xarray.open_dataset(
-    ic,
-    engine='ee',
-    projection=ic.first().select(0).projection(),
-    geometry=leg1
-)
-```
-
-Open multiple ImageCollections into one `xarray.Dataset`, all with the same
-projection:
+import ee, xarray as xr
```

Comment:
Might be worth the verbosity to have two separate import lines rather than `import ee, xarray as xr`:

```python
import ee
import xarray as xr
```